code/modules/library/lib_codex_gigas.dm

```diff
@@ -91,7 +92,7 @@
 	else if(GLOB.devil_suffix.Find(action))
 		currentSection = SUFFIX
 	else
-		to_chat(world, "Codex gigas failure [action]")
+		to_chat(world, "Codex Gigas failure [action]")
```

Comment:
Just remove this line and the else above it, There's a potential for anyone to spam the server with a codex gigas and a html request forger.